scripts/microbe_stage/microbe.lua

```diff
@@ -301,6 +305,55 @@ function Microbe:__init(entity)
     self.playerAlreadyShownVictory = false
 end
 
+-- Getter for microbe species
+-- 
+-- returns the species component or nil if it doesn't have a valid species
+function Microbe:getSpeciesComponent()
+    return Entity(self.microbe.speciesName):getComponent(SpeciesComponent.TYPE_ID)
+end
+
+
+--[[ deprecated, see SpeciesComponent:template
+-- Assigns a species to the microbe, if the species already exists
+-- If the species already exists, the microbe is updated to fit.
+-- Otherwise does nothing
+-- TODONE this is a very dirty function, and we should probably not do things this way
+-- deprecated, we now have SpeciesComponent:template to modify clean microbes
+function Microbe:setSpecies(speciesName)
+    self.microbe.speciesName = speciesName
+    if self.getSpeciesComponent() ~= nil then
+        -- update microbe
+        species = self.getSpeciesComponent()
+        -- remove all current organelles
+        self.microbe.organelles = {} -- we need to clear a bunch more data than this
+        for i, orgdata in pairs(species.organelles) do
+            organelle = OrganelleFactory.makeOrganelle(orgdata)
+            addOrganelle(orgdata.q, orgdata.r, organelle)
+        end
+    end
+end
+--]]
+
+--[[ deprecated, see SpeciesComponent:fromMicrobe
+function Microbe:makeSpecies(speciesName)
+    self.microbe.speciesName = speciesName
+    if not pcall(function() self.getSpeciesComponent() end) then
+        -- make species
+        speciesEntity = Entity(speciesName)
+        species = SpeciesComponent(speciesName)
+        speciesEntity:addComponent(species)
+        -- Create species' organelle data
+        for i, organelle in ipairs(self.microbe.organelles) do
+            print(i)
+            local data = {}
+            data.name = organelle.name
+            data.q = organelle.position.q
+            data.r = organelle.position.r
+            species.organelles[i] = data
+        end
+        for i,org in ipairs(species.organelles) do print(org.name,org.q,org.r) end
+    end
+end --]]
```

Comment:
Heh these are functions I added then removed, and that was a note-to-self to explain the removal